Lib/getpass.py

```diff
@@ -151,6 +169,33 @@ def _raw_input(prompt="", stream=None, input=None):
     return line
 
 
+def _input_with_echochar(prompt="", stream=None, input=None, echochar=""):
```

Comment:
this function is only called with all its parameters, so there is no need for defaults (and we also don't want defaults as `echochar=""` is not expected)